### ISSUE TITLE
[BC break] Resetting a non-lazy manager service is not supported

### ISSUE THREAD

**Issue by ro0NL** [doctrine/DoctrineBundle#1105]
> Updating doctrine/doctrine-bundle (1.12.2 => 1.12.4): Loading from cache

most of our tests now break with

```
LogicException: Resetting a non-lazy manager service is not supported. Declare the "doctrine.orm.default_entity_manager" service as lazy.
```

not much more info for now, but i wanted to report already.

If you need more info, let me know and i'll dive in :)

**Comment by stof** [comment]
@nicolas-grekas should we wire the resettable interface on the registry only when proxy-manager is installed ?

**Comment by alcaeus** [comment]
Fixed in 1.12.5 and 2.0.5

**Comment by ro0NL** [comment]
amazing :}
